src/main/java/trivia/game/PlayerManagement.java

```diff
@@ -0,0 +1,43 @@
+package trivia.game;
+
+import trivia.models.Player;
+import java.util.ArrayList;
+import java.util.List;
+
+public class PlayerManagement {
+
+    private Player active;
+    private long id = 1;
```

Comment:
id sounds like a misleading name, it doesn't communicate its intent well.
Is it about counting total number of Players in a round?